src/schema/compatibility.ts

```diff
@@ -214,15 +215,21 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         }
       },
     ),
-    bookmarks: compatFeedResolver((ctx, args, opts, builder, alias) =>
-      bookmarksFeedBuilder(
-        ctx,
-        false,
-        null,
-        builder.orderBy('bookmark.createdAt', 'DESC'),
-        alias,
-      ),
-    ),
+    bookmarks: async (source, args, context: AuthContext, info) => {
```

Comment:
What is this even used for? 😅

Reply:
Not entirely sure, but I think it is for backwards compat for way waaay back.